File: app/src/main/java/com/dev/lokeshkalal/creditCard/ValidatorErrorResponseCode.java
package com.dev.lokeshkalal.creditCard;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@IntDef({ValidatorErrorResponseCode.INVALID_CARD, ValidatorErrorResponseCode.INVALID_LENGTH})
@Retention(RetentionPolicy.SOURCE)
public @interface ValidatorErrorResponseCode {

    int INVALID_CARD = 2;
    int INVALID_LENGTH = 3;
}

File: app/src/main/java/com/dev/lokeshkalal/creditCard/addCreditCard/ValidatorErrorResponseCode.kt
package com.dev.lokeshkalal.creditCard.addCreditCard

import androidx.annotation.IntDef

import java.lang.annotation.RetentionPolicy

@IntDef(ValidatorErrorResponseCode.INVALID_CARD, ValidatorErrorResponseCode.INVALID_LENGTH)
@Retention(AnnotationRetention.SOURCE)
annotation class ValidatorErrorResponseCode {
    companion object {
        const val INVALID_CARD = 2
        const val INVALID_LENGTH = 3
    }
}
